xds/src/main/java/io/grpc/xds/sds/XdsChannelBuilder.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds.sds;
+
+import io.grpc.ForwardingChannelBuilder;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.grpc.netty.NettyChannelBuilder;
+import io.grpc.xds.sds.internal.SdsProtocolNegotiators;
+import java.net.SocketAddress;
+import javax.annotation.CheckReturnValue;
+
+/**
+ * A wrapper around NettyChannelBuilder that sets up the appropriate ProtocolNegotiator
+ * and ChannelHandlers to use SDS for TLS.
+ */
+public class XdsChannelBuilder
+    extends ForwardingChannelBuilder<XdsChannelBuilder> {
+
+  private final NettyChannelBuilder delegate;
+
+  private XdsChannelBuilder(NettyChannelBuilder delegate) {
+    this.delegate = delegate;
+    SdsProtocolNegotiators.setProtocolNegotiatorFactory(delegate);
+  }
+
+  /**
+   * Creates a new builder with the given server address.
+   * See {@link NettyChannelBuilder#forAddress(SocketAddress)} for more info.
```

Comment:
FYI. it is unclear why this is 2nd line. if you want the javadoc look like this, consider using `<p>`. if not please ignore.

```java
/** 
 * Create....
 *
 * <p>See ...
 */
```

Reply:
No the 2nd line is just for convenience (to avoid breaking at some other place)

Reply:
the reformat tool took care of this